packages/components/bucket-policy/bucket-policy-body.tsx

```diff
@@ -0,0 +1,442 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import classNames from 'classnames';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import {
+  Checkbox,
+  Divider,
+  Form,
+  FormGroup,
+  TextInput,
+  Select,
+  SelectOption,
+  SelectVariant,
+  Button,
+  Radio,
+} from '@patternfly/react-core';
+import { DataResourceType, EDIT_DATA_RESOURCES } from '../../constants';
+import {
+  NooBaaNamespaceStoreModel,
+  ProjectModel,
+  NooBaaObjectBucketClaimModel,
+} from '../../models';
+import { NamespaceStoreKind, K8sResourceKind } from '../../types';
+import { referenceForModel } from '../../utils';
+import { GenericDropdown } from '../../utils/dropdown/GenericDropdown';
+import { StatusBox } from '../../utils/generics/status-box';
```

Comment:
GenericDropdown also can be moved inside generic

Reply:
we already have separate folder for all dropdowns...